cmd/wavelet/server/server.go

```diff
@@ -0,0 +1,243 @@
+package server
```

Comment:
I think it's better to name the package `node` instead of `server`, since we're dealing with peer-to-peer instead of client-server.